components/ProgressView/src/MDCProgressView.m

```diff
@@ -29,6 +29,9 @@
 
 static const NSTimeInterval MDCProgressViewAnimationDuration = 0.25;
 
+static const CGFloat kLoadingBarWidth = 200;
```

Comment:
Should this always be 200? Should it not be some minimum/maximum value based on the bounds?